crates/llm-base/src/inference_session.rs

```diff
@@ -853,3 +853,36 @@ pub fn feed_prompt_callback<'a, E: std::error::Error + 'static>(
         None => Ok(InferenceFeedback::Continue),
     }
 }
+
+/// Callback to be passed to [InferenceSession::infer] that will print the
```

Comment:
Can you update this comment to better describe what the function does/how it's different to the normal callback?

Reply:
Updated!